jetstream/outcomes/firefox_desktop/networking_performance.toml

```diff
@@ -0,0 +1,86 @@
+friendly_name = "Networking Performance"
+description = "Network layer performance metrics"
```

Comment:
I'd suggest something that indicates that this is a subset of the other outcome.
```suggestion
description = "Network layer performance metrics (subset of Networking outcome without JS performance metrics)"
```

Reply:
Perfect, I've updated it with a comment regarding the subseting.